### PR TITLE
Feature/SC-8219 - Datetime correction on students consent page

### PR DESCRIPTION
**Pull request by murat-merdoglu-hpi**
# Description

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-8219

## Changes

- Adding a function in datetime.js file and some code-refactoring

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: src/plugins/datetime.js [hunk 1]
```diff
@@ -125,6 +125,19 @@ export const inputDateFromDeUTC = (date) => {
...
 };
 
+/**
+ * Returns formated date string based on a given date string in Backend format
+ * @param {String} date formatted date string from input fields (YYYY-MM-DD)
+ * @return {String} Date string based on current timezone for Backend format (DD.MM.YYYY)
+ */
+export const inputDateFormat = (date) => {
+	if (date) {
+		const result = dayjs(date, "YYYY-MM-DD").tz();
```
Comment (j-jmnz) [inline src/plugins/datetime.js:135]: Please use `DATETIME_FORMAT` variables instead of this string

File: src/plugins/datetime.js [hunk 2]
```diff
@@ -125,6 +125,19 @@ export const inputDateFromDeUTC = (date) => {
...
+		return result.format("DD.MM.YYYY");
```
Comment (j-jmnz) [inline src/plugins/datetime.js:136]: Same as above :)

File: src/plugins/datetime.js (hunk 1 above)
Comment (murat-merdoglu-hpi) [reply src/plugins/datetime.js:135]: Line 135 is changed to `DATETIME_FORMAT` variable.

File: src/plugins/datetime.js (hunk 2 above)
Comment (murat-merdoglu-hpi) [reply src/plugins/datetime.js:136]: Line 136: When a user changes the language from "de" to "en" the result will be "DD/MM/YYYY" format. It is not acceptable for the backend. So this line is not good to use `DATETIME_FORMAT` variables.

**Review by j-jmnz — APPROVED** [review]

**Pull request merged**